extensions/resource-deployment/src/ui/deployClusterWizard/deployClusterWizardModel.ts

```diff
@@ -166,16 +166,13 @@ export class DeployClusterWizardModel extends Model {
 			statements.push(`os.environ["DOCKER_PASSWORD"] = os.environ["${VariableNames.DockerPassword_VariableName}"]`);
 		}
 		const kubeCtlEnvVarName: string = getRuntimeBinaryPathEnvironmentVariableName(KubeCtlToolName);
-		statements.push(`os.environ["${kubeCtlEnvVarName}"] = "${this.escapeForNotebookCodeCell(process.env[kubeCtlEnvVarName]!)}"`);
-		statements.push(`os.environ["PATH"] = os.environ["PATH"] + "${delimiter}" + "${this.escapeForNotebookCodeCell(process.env[ToolsInstallPath]!)}"`);
+		const env: NodeJS.ProcessEnv = {};
+		setEnvironmentVariablesForInstallPaths(tools, env);
```

Comment:
is this really required? i thought we have set the tool path when we were checking the tools before the wizard is launched.

Reply:
yes because the tools should not be setting OS.environ["PATH"] which older code requires. I thought you wanted to remove reliance on setting environment variables except for passwords. With this change in place, tools installation and discovery do not need to set process.env[ToolsInstallPath] and process.env[kubeCtlEnvVarName] which the old code depends on. 

If you would rather that I not make this change, I can leave it as it is as this is just cleaning up some usage.

Reply:
The setting of toolspath in process.env has changed with the change to in this PR:
extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

We no longer set process.env variables for anything. only wizards and dialogtabs will set process.env variables and that too only for passwords.

Reply:
So I would like to keep this change unless you insist that we continue to keep the previous code.